master/getting-started/openstack/upgrade/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Upgrading Calico for OpenStack
```

Comment:
If you notice in the Kubernetes section, I don't have "for Kubernetes". Yet here, we have "for OpenStack." Should be consistent. I would lean towards omitting "for OpenStack" from these titles. If we feel like they are necessary, then we should make the Kubernetes headings match. 

Another comment should we feel like these are needed, I think that "on" is better than "for". again, this is something that should match up between the sections.

Reply:
"on" might be right for kubernetes where calico can actually run in a hosted fashion, but I'd argue it isn't right in Openstack.  We can discuss if you wish :)